test/alpha/beta/gamma/TEST

```diff
@@ -0,0 +1 @@
+These are test directories for create command's subdirectory (-s) flag
```

Comment:
you may want to write to the test/temp directory for this test since that dir is already in the .gitignore